src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -0,0 +1,640 @@
+        speciesPreview.PreviewSpecies = species;
+        hexPreview.PreviewSpecies = species as MicrobeSpecies;
+        UpdateSpeciesDetail(species);
+    }
+
+    private void EvolutionaryTreeNodeSelected(int generation, uint id)
+    {
+        HistoryListMenuIndexChanged(generation);
+        UpdateSpeciesPreview(speciesHistoryList[generation][id]);
+    }
+
+    private void UpdateSpeciesDetail(Species species)
+    {
+        /*
+        speciesDetailsLabel.ExtendedBbcode = $"[b]Species:[/b]\n  {species.FormattedNameBbCode}:{species.ID}\n" +
+            $"[b]Generation:[/b]\n  {species.Generation}\n" +
+            $"[b]Population:[/b]\n  {species.Population}\n" +
+            $"[b]Colour:[/b]\n  #{species.Colour.ToHtml()}\n" +
+            $"[b]Behaviour[/b]\n  {species.Behaviour.Join(b => b.Key + ": " + b.Value, "\n  ")}\n";
+
+        switch (species)
+        {
+            case MicrobeSpecies microbeSpecies:
+            {
+                speciesDetailsLabel.ExtendedBbcode += $"[b]Stage:[/b]\n  Microbe\n" +
+                    $"[b]Membrane Type:[/b]\n  {microbeSpecies.MembraneType.Name}\n" +
+                    $"[b]Membrane Rigidity:[/b]\n  {microbeSpecies.MembraneRigidity}\n" +
+                    $"[b]Base Speed:[/b]\n  {microbeSpecies.BaseSpeed}\n" +
+                    $"[b]Base Rotation Speed[/b]\n  {microbeSpecies.BaseRotationSpeed}\n" +
+                    $"[b]Base Hex Size[/b]\n  {microbeSpecies.BaseHexSize}";
+                break;
+            }
+        }
+        */
+
+        speciesDetailsLabel.ExtendedBbcode = string.Format(CultureInfo.CurrentCulture,
+            TranslationServer.Translate("SPECIES_DETAIL_TEXT"), species.FormattedNameBbCode, species.ID,
+            species.Generation, species.Population, species.Colour.ToHtml(),
+            species.Behaviour.Join(b => b.Key + ": " + b.Value, "\n  "));
+
+        switch (species)
+        {
+            case MicrobeSpecies microbeSpecies:
+            {
+                speciesDetailsLabel.ExtendedBbcode += string.Format(CultureInfo.CurrentCulture,
+                    TranslationServer.Translate("MICROBE_SPECIES_DETAIL_TEXT"), microbeSpecies.MembraneType.Name,
+                    microbeSpecies.MembraneRigidity, microbeSpecies.BaseSpeed, microbeSpecies.BaseRotationSpeed,
+                    microbeSpecies.BaseHexSize);
+                break;
+            }
+        }
+    }
+
+    private void PlayWithCurrentSettingPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        // Disable the button to prevent it being executed again.
+        playWithCurrentSettingButton.Disabled = true;
+
+        TransitionManager.Instance.AddSequence(ScreenFade.FadeType.FadeOut, 0.1f, () =>
+        {
+            // Instantiate a new editor scene
+            var editor = (MicrobeEditor)SceneManager.Instance.LoadScene(MainGameState.MicrobeEditor).Instance();
+
+            // Start freebuild game
```

Comment:
```suggestion
            // Copy our currently setup game to the editor
```

Reply:
Copied from freebuild editor (

Reply:
Well this is the wrong comment anyway to be in this context.

This is the code there for reference:
```
            // Start freebuild game
            editor.CurrentGame = GameProperties.StartNewMicrobeGame(new WorldGenerationSettings(), true);
```
How you are using the code is *completely* different, it doesn't make sense to keep the comment as is here as it is incorrect.